### PR TITLE
Clarification for non-electric aids during FMC.

### PR DESCRIPTION
**Pull request by hanwu85**
Addressing issue #815

### CODE REVIEW

**Review by Ivan-Brigidano — COMMENTED** [review]
This is clearer than #839 but still we should wait as technically this is not a clarification but a change in Regulations

**Comment by Ivan-Brigidano** [comment]
(Closing until 2020 Regulations are released. Will reopen if 2020 Regulations include it)

File: wca-guidelines.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ To be more informative, each Guideline is classified using one of the following
 - 3h++) [EXAMPLE] Examples of enhancements include: new moves are possible, normal moves are impossible, more pieces or faces are visible, colors on the backside of the puzzle are visible, moves are done automatically, or the puzzles have more/different solved states.
 - 3h2++) [CLARIFICATION] "Stickerless" puzzles that significantly differ from most mass-produced "stickerless" puzzles are only permitted at the discretion of the WCA Delegate.
 - 3j+) [REMINDER] In the past, engraved/embossed parts have been permitted. This is no longer permitted.
-- 3j++) [CLARIFICATION] On Clock, pins must not be distinguishable from any other pin of the same side.
+- 3j++) [CLARIFICATION] A clock with distinguishable pins from any other pin of the same side is permitted.
```
Comment (Ivan-Brigidano) [inline wca-guidelines.md]: This isn't the PR for Clock pins @hanwu85

**Review by Ivan-Brigidano — CHANGES_REQUESTED** [review]

File: wca-guidelines.md (hunk 1 above)
Comment (hanwu85) [reply wca-guidelines.md]: This is strange, I didn't change this one...Branch mixed up I think.

**Review by Ivan-Brigidano — APPROVED** [review]

**Pull request merged**